src/mcp_server_helm/server.py

```
@@ -1,1682 +1,27 @@
-        await server.run(read_stream, write_stream, options, raise_exceptions=True)
-
-# Add this main function at the end
 def main():
     """
     Entry point for the MCP Helm server when run as a command-line program.
     """
-    import asyncio
-
     logging.basicConfig(level=logging.INFO,
                         format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
     logger = logging.getLogger(__name__)
```

Pylint (3.3.7)

[warning] Redefining name 'logger' from outer scope (line 8)

(W0621)